tutorials/CommandsArguments.md

```diff
@@ -0,0 +1,64 @@
+# Commands I: Arguments
+
+This is an in-depth tutorial for **Klasa's Usage arguments**, explaining all kinds of arguments and what you can do with them, as well as how they're used in the framework's backends.
+
+## Command Arguments
+
+What are command arguments and what they differ with parameters? Command arguments are the *tags* or *possibles* that specifies what should the framework take and what not, such as the type and its required level, therefore, they are a very important part from the **internal parsing** from Klasa: the framework runs and validates the arguments, parsing them to **parameters**.
```

Comment:
This confuses 3 concepts: usage tags / possibles, arguments (they are only raw strings split based on the delim / quoted strings support), and resolvers which convert args to parameters